### DISCUSSION TITLE
problem with installment

### DISCUSSION THREAD

**Discussion by HybridTheGamerAndVideoMaker** [DarkflameUniverse/DarkflameServer#1093] [General]
i got it installed and unzipped but idk what to do since i am autistic. can u explain what i need to do?

**Comment by MustacheMan2** [comment]
I don't expect anyone here to hold your hand through setup of the server, which is why good guides exist. Assuming you are on Windows, see https://gist.github.com/HailStorm32/169df65a47a104199b5cc57d10fa57de

**Reply by HybridTheGamerAndVideoMaker** [reply]
DOWNLOAD A PROGRAM!? What!? not fair! I wanted to play the game, not go through stressful, hard or confusing set ups. Theres got to be a easier way for people with difficulties. i have trouble understanding certain things i heard or read because of my autism. i feel like this is a scam file!

**Comment by aronwk-aaron** [comment]
stale,closing

**Reply by HybridTheGamerAndVideoMaker** [reply]
no it not

**Reply by aronwk-aaron** [reply]
Your question has been answered. There are guides on how to setup the server linked above and in the readme of this repo. You must acquire a client from somewhere on the internet to be able to play as well. there are checksums provided in the repo to help you verify that you have a good client.
This is the current state of things and there is nothing that we or anyone can do about it unfortunately

**Reply by HybridTheGamerAndVideoMaker** [reply]
Ah man. there goes the chance to redeem those lego childhood memories.... :(
